<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtKalmanFilterDialog</class>
 <widget class="QDialog" name="QtKalmanFilterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>222</width>
    <height>106</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtKalmanFilterDialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QComboBox" name="box_filter_type">
     <item>
      <property name="text">
       <string>Standard Kalman filter</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Steady state Kalman filter</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_estimated_lag">
       <property name="text">
        <string>Estimated lag</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="box_estimated_lag">
       <property name="sizePolicy">
        <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="button_calculate_optimal_kalman_gain_steady_state">
     <property name="text">
      <string>Calculate optimal Kalman gain
(from standard Kalman filter parameters)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
